contents/blog/how-we-built-an-app-server.md

```diff
@@ -23,11 +23,11 @@ This is the story of how [PostHog apps](/apps) went from three-day MVP into a se
 
 PostHog apps are the brainchild of our [first ever hire](/blog/posthog-first-five), Marius Andra. 
```

Comment:
This intro doesn't quite set up this article to be read first person, which then then means you have that awkward 'I (Marius)' later

Reply:
rewrote intro to make first person from the start.

Reply:
Yeah that works 👍